git-sync/ssh.md

```diff
@@ -0,0 +1,64 @@
+# Using SSH with git-sync
+
+Git-sync supports using the SSH protocol for pulling Git content.
+
+For now, only keys without password protection are supported.
```

Comment:
Yeah I don't really know how password insertion would work, but it's my understanding that some SSH private keys are password-protected, and I just wanted to put it out there that those aren't going to work. I suppose I should get rid of "For now".